scripts/library_tools.py

```
@@ -0,0 +1,839 @@
+        index = yaml.safe_load(index_path.read_text()) or {"elements": []}
+    else:
+        index = {"elements": []}
+
+    entry = {
+        "file": target_file,
+        "subject": args.subject,
+        "style": args.style,
+        "style_ref": args.style_ref or "",
+        "provider": args.provider,
+        "style_id": args.style_id or "",
+        "generated_at": str(date.today()),
```

Ruff (0.15.12)

[warning] `datetime.date.today()` used

(DTZ011)